src/app/common/validation/forms/amount-validators.ts

```diff
@@ -32,50 +40,60 @@ export function btcInsufficientBalanceValidator({
   recipient,
   calcMaxSpend,
 }: BtcInsufficientBalanceValidatorArgs) {
-  return yup.number().test({
-    message: 'Insufficient funds',
-    test(value) {
-      if (!value) return false;
-      const maxSpend = calcMaxSpend(recipient);
-      if (!maxSpend) return false;
-      const desiredSpend = new BigNumber(value);
-      if (desiredSpend.isGreaterThan(maxSpend.spendableBitcoin)) return false;
-      return true;
-    },
-  });
+  return yup
+    .number()
+    .typeError(formatErrorWithSymbol('BTC', FormErrorMessages.MustBeNumber))
+    .test({
+      message: FormErrorMessages.InsufficientFunds,
+      test(value) {
+        if (!value) return false;
+        const maxSpend = calcMaxSpend(recipient);
+        if (!maxSpend) return false;
+        const desiredSpend = new BigNumber(value);
+        if (desiredSpend.isGreaterThan(maxSpend.spendableBitcoin)) return false;
+        return true;
+      },
+    });
 }
 
 export function btcMinimumSpendValidator() {
-  return yup.number().test({
-    message: `Minimum is ${satToBtc(minSpendAmountInSats)}`,
-    test(value) {
-      if (!value) return false;
-      const desiredSpend = btcToSat(value);
-      if (desiredSpend.isLessThan(minSpendAmountInSats)) return false;
-      return true;
-    },
-  });
+  return yup
+    .number()
+    .typeError(formatErrorWithSymbol('BTC', FormErrorMessages.MustBeNumber))
```

Comment:
Do we need to repeat this? Could set once in the place where we compose them together?

Reply:
Let me check, but def needs it if using `.number()` or you get the default type error in UI shown in issue linked here.

Reply:
I know we need the type error, I just wonder if we can't not repeat it. Does it work to create a separate 

```
btcAmountTypeError() {
  return yup.number().typeError(formatErrorWithSymbol('BTC', FormErrorMessages.MustBeNumber))
}
```

and then we compose that too like the others?

Reply:
Moving it there does not appear to work.